### DISCUSSION TITLE
Ability to change default width, size and scale for remote image rendering

### DISCUSSION THREAD

**Discussion by laundmo** [grafana/grafana#47211] [Dashboards, Panels & Visualizations]
Currently, the default width/height is hard-coded to 1000x500 here: https://github.com/grafana/grafana/blob/f1c3177e79f21e1292afa02efa393f5e52dc231a/public/app/features/dashboard/components/ShareModal/utils.ts#L106

The reason i set up the image renderer in the first place was to produce higher quality images than screenshotting would give me. Having to edit the URL by hand for this is really not great.

I propose being able to set a default server-wide for these parameters, and down the line in the future making them easily configurable in the ShareModal. Especially, being able to set the `scale` parameter is essential to take full advantage of the image renderer.

**Comment by SamuelToh** [comment]
I suggest adding couple of labels and textboxes beside the 'Direct link rendered image' button. Default value will be 1000 by 500 and the button will construct the url according to the input values.

![image](https://user-images.githubusercontent.com/8707446/208592043-43539ff0-74b9-4b40-81fd-af3cc28e4f9e.png)

**Comment by Huangvivi** [comment]
any update here?

**Comment by danielthedm** [comment]
Moved to issue: https://github.com/grafana/grafana/issues/75963
